src/Uno.UI.BindingHelper.Android/Uno/UI/UnoMotionHelper.java

```diff
@@ -25,16 +25,20 @@ protected MotionTargetAdapter(Uno.UI.UnoMotionTarget target) {
 	final boolean getCustomDispatchIsActive() { return _isCustomDispatchIsActive; }
 	final void setCustomDispatchIsActive(boolean isCustomDispatchIsActive) { _isCustomDispatchIsActive = isCustomDispatchIsActive; }
 
-	private View _customDispatchTarget;
-	final View getCustomDispatchTouchTarget() { return _customDispatchTarget; }
-	final void setCustomDispatchTouchTarget(View child) {
-		if (getIsTargetCachingSupported()) {
-			_customDispatchTarget = child;
-		}
-	}
+	// The "cached target" is used by the custom dispatch logic to track the last child to which an event was sent
+	// This is required in order to make sure to follow the full events sequences: down->move->up OR enter->move->exit
+	// (For instance on a HOVER_MOVE, if the pointer goes out of the current "cached target" we still make sure to
+	// send it to this "cached target" the event in order to raise the HOVER_EXIT)
+	private View _cachedTarget;
+	final View getCachedTarget() { return _cachedTarget; }
```

Comment:
Cached means "current" ?

Reply:
Kind of, as this "current" is relative to a given sequence (down / move / up **or** hover enter / move / exit) and it might be cleared depending of various factor (for instance when moving pointer over a sibling). I preferred the name "cached" as it really just to avoid re-calculation in that case ... but meh 🤷‍♂️ ...

Reply:
I will rename it in my next PR (multi touch) as I also change this file.